test/asn1_internal_test.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 1999-2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* Internal tests for the asn1 module */
+
+#include <stdio.h>
+#include <string.h>
+
+#include <openssl/asn1.h>
+#include <openssl/evp.h>
+#include <openssl/objects.h>
+#include "testutil.h"
+#include "e_os.h"
+
+typedef struct {
+    const char *test_case_name;
+    const char *test_section;
+} SIMPLE_FIXTURE;
+
+/**********************************************************************
+ *
+ * Test of a_strnid's tbl_standard
+ *
+ ***/
+
+static SIMPLE_FIXTURE setup_tbl_std(const char *const test_case_name)
```

Comment:
I'll meet you half way and rename `tbl_std` -> `tbl_standard`, since the latter is the actual variable name.